README.md

```diff
@@ -173,11 +173,15 @@ grunt watch
 Run the unit tests with a local server that supports PHP. Ensure that you run the site from the root directory, not the "test" directory. No database is required. Pre-configured php local servers are available for Windows and Mac. Here are some options:
 
 - Windows: [WAMP download](http://www.wampserver.com/en/)
-- Mac: [MAMP download](http://www.mamp.info/en/index.html)
 - Linux: [Setting up LAMP](https://www.linux.com/learn/tutorials/288158-easy-lamp-server-installation)
 - [Mongoose (most platforms)](http://code.google.com/p/mongoose/)
+- Mac: [MAMP download](http://www.mamp.info/en/index.html)
```

Comment:
I shouldn't have committed this change. There is nothing different compared
to before.

I initially thought about altering the Mac installation description for
Lamp, because PHP was already installed on OS X.

Then I thought it would be better just to insert a quick tip for using PHPs
build server if you're on OS X or already have OS X installed.
Because maybe some people still want to use LAMP on OS X.

On Monday, August 17, 2015, Michał Gołębiowski <notifications@github.com>
wrote:

> In README.md
> <https://github.com/jquery/jquery/pull/2537#discussion_r37152489>:
>
> >  - Linux: [Setting up LAMP](https://www.linux.com/learn/tutorials/288158-easy-lamp-server-installation)
> >  - [Mongoose (most platforms)](http://code.google.com/p/mongoose/)
> > +- Mac: [MAMP download](http://www.mamp.info/en/index.html)
>
> Why this change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/jquery/pull/2537/files#r37152489>.
>